presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -117,7 +117,7 @@ public void testMySqlCreatedParameterizedVarchar()
                 .addRoundTrip(stringDataType("mediumtext", createVarcharType(16777215)), "c")
                 .addRoundTrip(stringDataType("longtext", createUnboundedVarcharType()), "d")
                 .addRoundTrip(varcharDataType(32), "e")
-                .addRoundTrip(varcharDataType(20000), "f")
+                .addRoundTrip(varcharDataType(15000), "f")
```

Comment:
Is this related to `Update testing-mysql-server to 8.0.12` ?

Reply:
Yes, I tried the original and it fails with

> Column length too big for column 'col_5' (max = 16383); use BLOB or TEXT instead